### PR TITLE
fix(breakout-rooms) Improve breakout rooms tests

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

**Comment by damencho** [comment]
To avoid the stale element error, obtain the roomlist object before checking it, wdyt?

**Review by saghul — DISMISSED** [review]
LGTM with a question.

**Comment by damencho** [comment]
We discussed it offline and decided on a new approach as this one is not changing a lot from previous behavior

File: src/test/java/org/jitsi/meet/test/web/WebParticipant.java [hunk 1]
```diff
@@ -648,12 +648,7 @@ public KnockingParticipantList getKnockingParticipantList()
      */
     public BreakoutRoomsList getBreakoutRoomsList()
     {
-        if (breakoutRoomsList == null)
```
[the remark this answers is not in the document]
Comment (robertpin) [reply src/test/java/org/jitsi/meet/test/web/WebParticipant.java]: Nvm that :)

**Review by damencho — APPROVED** [review]

**Pull request merged**